src/api/app/assets/stylesheets/webui/diff.scss

```diff
@@ -31,7 +38,8 @@
         left: 0;
         .number {
           color: $text-muted;
-          width: 4rem;
+          font-family: monospace;
+          width: calc(var(--line-index-digits) + 2rem + #{$card-border-width});
```

Comment:
Would it not be simpler to set `max-width` to a certain amount of characters? The content would then take the space it needs, up to that `max-width` value.

Reply:
We talked and it is not possible.